Fix login and a few smaller issues (#21)
SUSE-Developer-Community/sandbox-welcome-content: content/cli.md

Once you have installed the cf-cli, we need to log in. In this sandbox environment, we need to log in though our Single Sign On portal. To start the sign on, run:

```bash
cf login -a < TODO  url> -u <Email used in developer portal> 
```

> This will prompt you for your password. This is the random password delivered along with your welcome email. (If you have since changed your password on Stratos, use what you changed it to)

Reviewer: ```suggestion
This will prompt you for your password. This is the random password delivered along with your welcome email. Remember to use your updated password in case you changed it in Stratos. 
```

> Regardless of which language your write your app in, the last few lines of the output should look something like this:

Reviewer: ```suggestion
Regardless of which language you write your app in, the last few lines of the output should look something like this:
```
